riscv-asm.md

```diff
@@ -397,9 +397,10 @@ linker relaxation accidentally if user already disable linker relaxation.
 
 Push/pop current options to/from the options stack.
 
-## Assembler Relocation Functions
+## Assembler Modifiers
```

Comment:
This term is learned from GNU as manual https://sourceware.org/binutils/docs/as/RISC_002dV_002dModifiers.html